README.md

```diff
@@ -1,11 +1,11 @@
 # Neural Net Demo for TDI
-30 second introduction to ML with neural nets.
+This is a quick introduction to ML with neural nets.
 
-Comparision of tensorflow with keras.
```

Comment:
One of the big points here is that Keras has a much lower development time than tensorflow and gives comparable results.